sdk/python/packages/flet-core/src/flet_core/alert_dialog.py

```diff
@@ -73,11 +75,21 @@ def __init__(
         title_padding: PaddingValue = None,
         content_padding: PaddingValue = None,
         actions_padding: PaddingValue = None,
-        actions_alignment: MainAxisAlignment = MainAxisAlignment.NONE,
+        actions_alignment: MainAxisAlignment = None,
```

Comment:
If we don't need `NONE` in the enum then it should be `Optional[MainAxisAlignment]` here.

Reply:
...and remove `NONE` from enum then.